requirements.txt

```diff
@@ -1,2 +1,6 @@
 ovos-plugin-manager>=0.0.5
-piper-tts==1.2.0
+ovos-utils
```

Comment:
I believe this PR enforces a newer version of ovos-utils, due to the lang functions you pull in